doc-content/shared-kie-docs/src/main/asciidoc/KieServer/securing-passwords-proc.adoc

```diff
@@ -0,0 +1,185 @@
+[source,bash]
+----
+$<EAP_HOME>/add-user.sh -a -e -u <USER> -p "<PASSWORD>" -g kie-server
+----
+endif::[]
+ifeval::["{context}" == "install-on-jws"]
+[source,bash]
+----
+$<JWS_HOME>/add-user.sh -a -e -u <USER> -p "<PASSWORD>" -g kie-server
+----
+endif::[]
+ifeval::["{context}" == "install-on-tomcat"]
+[source,bash]
+----
+$<TOMCAT_HOME>/add-user.sh -a -e -u <USER> -p "<PASSWORD>" -g kie-server
+----
+endif::[]
+ifeval::["{context}" == "dsps-on-was"]
+[source,bash]
+----
+$<WAS_HOME>/add-user.sh -a -e -u <USER> -p "<PASSWORD>" -g kie-server
+----
+endif::[]
+ifeval::["{context}" == "dsps-on-wls"]
+[source,bash]
+----
+$<WLS_HOME>/add-user.sh -a -e -u <USER> -p "<PASSWORD>" -g kie-server
+----
+endif::[]
+endif::[]
++
+. To use KeyTool to create a JCEKS, enter the following command in the Java 8 home directory:
++
+[source,bash]
+----
+$<JAVA_HOME>/bin/keytool -importpassword -keystore <KEYSTORE_PATH> -keypass <ALIAS_KEY_PASSWORD> -alias <PASSWORD_ALIAS> -storepass <KEYSTORE_PASSWORD> -storetype JCEKS
+----
++
+In this example, replace the following variables:
+
+* `<KEYSTORE_PATH>`: The path where the keystore will be stored
+* `<KEYSTORE_PASSWORD>`: The keystore password
+* `<ALIAS_KEY_PASSWORD>`: An alias used to store passwords
+* `<PASSWORD_ALIAS>`: Password associated with `<ALIAS_KEY_PASSWORD>`
+
+. When prompted, enter the password for the {KIE_SERVER} user that you created.
+. Set the following system properties in the
+ifdef::DROOLS,JBPM,OP[]
+`WILDFLY_HOME/standalone/configuration/standalone-full.xml`
+endif::[]
+ifdef::DM,PAM[]
+ifeval::["{context}" == "install-on-eap"]
+`EAP_HOME/standalone/configuration/standalone-full.xml`
+endif::[]
+ifeval::["{context}" == "install-on-jws"]
+`JWS_HOME/standalone/configuration/standalone-full.xml`
+endif::[]
+ifeval::["{context}" == "install-on-tomcat"]
+`TOMCAT_HOME/standalone/configuration/standalone-full.xml`
+endif::[]
+ifeval::["{context}" == "dsps-on-was"]
+`WAS_HOME/standalone/configuration/standalone-full.xml`
+endif::[]
+ifeval::["{context}" == "dsps-on-wls"]
+`WLS_HOME/standalone/configuration/standalone-full.xml`
+endif::[]
+endif::[]
+ file and replace the placeholders as listed in the table that follows:
++
+[source,xml]
+----
+    <system-properties>
+        <property name="kie.keystore.keyStoreURL" value="<PATH>">
+        <property name="kie.keystore.key.ctrl.pwd" value="<KEYSTORE_PWD>"/>
+        <property name="kie.keystore.key.server.alias" value="<KEY_SERVER_ALIAS>"/>
+        <property name="kie.keystore.key.server.pwd" value="<KEY_SERVER_PWD>"/>
+        <property name="kie.keystore.key.ctrl.alias" value="<KEY_CONTROL_ALIAS>"/>
+        <property name="kie.keystore.key.ctrl.pwd" value="<KEY_CONTROL_PWD>"/>
+    </system-properties>
+----
++
+.System properties used to load a {KIE_SERVER} JCEKS
+[cols="40%,30%,40%", frame="all", options="header"]
+|===
+| System property
+| Placeholder
+| Description
+
+| `kie.keystore.keyStoreURL`
+| `<PATH>`
```

Comment:
This should be KEYSTORE_URL, please adjust also  <property name="kie.keystore.keyStoreURL" value="<PATH>">